includes/base_controls/_events.inc.php

```diff
@@ -328,13 +328,13 @@ class QOnEvent extends QEvent{
 		/**
 		 * Constructor
 		 * @param int  $strEventName
-		 * @param string $strSelector
 		 * @param string $strCondition
 		 * @param int  $intDelay
+		 * @param string $strSelector
 		 *
 		 * @throws Exception|QCallerException
 		 */
-		public function __construct($strEventName, $strSelector = null, $strCondition = null, $intDelay = 0) {
+		public function __construct($strEventName, $strCondition = null, $intDelay = 0, $strSelector = null) {
```

Comment:
Because I am too tired to do this. Thanks for catching this. Will fix it in a bit.